packages/react-instantsearch-hooks-web/src/ui/ShowMoreButton.tsx

```diff
@@ -9,11 +9,9 @@ export type ShowMoreButtonTranslations = {
   /**
    * Alternative text for the show more button.
    */
-  showMoreButtonText: string;
-  /**
-   * Alternative text for the show less button.
-   */
-  showLessButtonText: string;
+  showMoreButtonText: (showMoreButtonTextOptions: {
```

Comment:
```suggestion
  showMoreButtonText: (state: {
```

```suggestion
  showMoreButtonText: (options: {
```

```suggestion
  showMoreButtonText: (data: {
```

It may make more sense being the same for every translation

Reply:
Yeah I changed it to options later, same for Pagination

Reply:
`data` > `options` IMO but it's no big deal since this isn't a named one and we can change it whenever 🙂